style_editor.html

```diff
@@ -10291,6 +10296,21 @@ <h4 class="settings-section-label" title="These settings apply to the blade prev
   body.classList.toggle("dark-mode", on);
   structuredView.classList.toggle("dark-mode", on);
 });
+var tipsState = new SavedState("tips", true, (on) => { 
+ if (on) {
+    const elementsWithDataTitles = document.querySelectorAll("[data-title]");
+    elementsWithDataTitles.forEach((element) => {
+      element.setAttribute("title", element.dataset.title);
+      element.removeAttribute("data-title");
+    });
+  } else {
+    const elementsWithTitles = document.querySelectorAll("[title]");
+    elementsWithTitles.forEach((element) => {
+      element.dataset.title = element.getAttribute("title");
+      element.removeAttribute("title");
```

Comment:
Wouldn't it be easier to just make some change in the CSS?
Seems heavy to go change every element.
Also, don't we make element with titles in the code sometimes?

Reply:
I would debate "easier". I would think with the negligible hit possibly taken on this method, it avoids cross browser and device incompatibility, avoids having to edit a lot of existing HTML, and is tested extensively already.
Currently, my opinion is this is the most straight forward, sensical way.

As far as "making elements with titles", that is likely the Tabs content, which this currently does not accommodate. I was hoping to lean on you a bit for that, as you had once mentioned you could do that rather easily.
At minimal, it would be a separate PR.

The dynamically created elements are something that still elude me as to how you have them working exactly, in addition to the history and "MAGIC_CLASS" etc...
How could we get the string from the "super" line of code in the classes to make them become the title attribute of the created element?

Reply:
Doesn't the structured view have titles?
If not, it probably should.

Reply:
It does. It seems like a lot of modification to AddTab, mkbutton2, EnumBuilder etc... to get the tab buttons to have title attributes. 
Again, that all feels like a separate PR, as I also have relevant tabs enabling/disabling "up my sleeve"

Reply:
I think it would be better if we could have some way to enable/disable the popups without modifying the DOM.
It's kind of a pain to keep track of whether we should set "title" or "data.title" when we generate code.

Reply:
Although, I tried reading up on possible solutions, and I didn't really find any.
Looks like we'll actually need to modify the DOM :(

Reply:
data-title is never used, so it is a good temp holding area?
New elements would just get titles like everything else without confusion.